clients/src/main/java/org/apache/kafka/clients/ClientTelemetryReporter.java

```diff
@@ -550,6 +576,41 @@ public void close() {
             }
         }
 
+        @Override
+        public void initiateClose(long timeoutMs) {
+            log.debug("initiate close for client telemetry, check if terminal push required. Timeout {} ms.", timeoutMs);
+
+            lock.writeLock().lock();
+            try {
+                // If we never fetched a subscription, we can't really push anything.
+                if (lastRequestMs == 0) {
+                    log.info("Telemetry subscription not loaded, not attempting terminating push");
```

Comment:
This is `info` while others below are `debug`. Why?

Reply:
Missed it, my bad, earlier kept everything at INFO for local testing. Moved to debug.